OloEngine/tests/Rendering/CommandDispatchTest.cpp

```
@@ -0,0 +1,190 @@
+#include "OloEngine/Renderer/Commands/CommandDispatch.h"
+#include "OloEngine/Renderer/Commands/CommandBucket.h"
+#include "OloEngine/Renderer/Commands/CommandAllocator.h"
+
+using namespace OloEngine;          // NOLINT(google-build-using-namespace)
+using namespace OloEngine::Testing; // NOLINT(google-build-using-namespace)
+
+// =============================================================================
+// Dispatch Table Completeness
+// =============================================================================
+
+TEST(CommandDispatch, DispatchTableIsComplete)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)